### PR TITLE
changed store results to reflect using user location or not

### PR DESCRIPTION
**Pull request by SamDaltonJr**

### CODE REVIEW

File: src/components/StoreResult/StoreResult.js [hunk 1]
```diff
@@ -17,35 +17,53 @@ export class StoreResult extends React.Component {
...
+          <strong>{name}</strong>
+          <address>
+            {location.address1} {location.address2}
+          </address>
+          <p>
+            <DistanceDisplay meters={store.distance} /> away
+          </p>
+        </li>
+      );
+    } else {
```
Comment (aseehra) [inline src/components/StoreResult/StoreResult.js]: The `else` is technically not necessary because you return in the `if` above.

**Comment by aseehra** [comment]
Additionally, the scroll bar is still visible even though you're only sending 7 items now.

File: src/components/StoreSearch/StoreSearch.js [hunk 2]
```diff
@@ -25,6 +28,24 @@ export class StoreSearch extends React.Component {
     if (this.props.error) {
       return;
     }
+    if (this.props.loading) {
+      return (
+        <div class="lds-spinner">
```
[the remark this answers is not in the document]
Comment (SamDaltonJr) [reply src/components/StoreSearch/StoreSearch.js]: Sure can

[the remark this answers is not in the document]
Comment (aseehra) [reply src/components/StoreSearch/StoreSearch.js]: :heart:

**Pull request merged**